src/BalanceClient.ts

```diff
@@ -0,0 +1,22 @@
+const {providers} = require("near-api-js");
+
+export class BalanceClient {
+
+  static async balanceOfOnNear(contractId: string, accountId: string): Promise<number> {
+    const connectionInfo = { url: "https://rpc.testnet.near.org" };
+    const provider = new providers.JsonRpcProvider(connectionInfo);
+
+    const args = `{ "account_id": "${accountId}" }`;
+    const encodedArgs = Buffer.from(args).toString("base64");
+
+    const queryResponse = await provider.query({
+      request_type: "call_function",
+      account_id: contractId,
+      method_name: "ft_balance_of",
+      args_base64: encodedArgs,
+      finality: "final",
+    });
+
+    return JSON.parse(Buffer.from(queryResponse.result).toString());
+  }
+}
```

Comment:
lets also add `nft_balance_of`-like methods to check nft ownership per address on near and calimero

Reply:
This is fine for now, we can also add `nft_tokens_for_owner` later 
https://nomicon.io/Standards/Tokens/NonFungibleToken/Enumeration

Reply:
I added this already in this PR since I thought it might be helpful even for simple use cases.